docs/quickstart/installation.md

```diff
@@ -60,16 +60,16 @@ You can build {{ $themeConfig.project.name }} using Docker by running:
 make build-docker
 ```
 
-The command above will create a docker container: `tharsis/evmos:latest`. Now you can run `evmosd` in the container.
+The command above will create a docker container: `tharsislabs/evmos:latest`. Now you can run `evmosd` in the container.
```

Comment:
`tharsis` uname in docker hub was already taken

Reply:
what about tharsishq?